pkg/notify/notify.go

```diff
@@ -60,7 +60,12 @@ func (r *Receiver) signalNonBlocking() bool {
 func (r *Receiver) signalTickLoop() {
 	go func() {
 	loop:
-		for range r.ticker.C {
+		for {
+			select {
+			case <-r.closeCh:
+				break
+			case <-r.ticker.C:
+			}
```

Comment:
Does it cause performance regression?

Reply:
This won't be a performance bottleneck here theoretically.